draft-dekater-scion-controlplane.md

```diff
@@ -1423,15 +1423,16 @@ An endpoint (source) that wants to start communication with another endpoint (de
   - a core AS in a remote ISD, if the destination AS is in another ISD, and
 - a down-path segment to reach the destination AS.
 
-**Note:** The actual number of required path segments depends on the location of the destination AS as well as on the availability of shortcuts and peering links. More information on combining and constructing paths is provided by {{I-D.scion-dp}}.
+The actual number of required path segments depends on the location of the destination AS as well as on the availability of shortcuts and peering links. More information on combining and constructing paths is provided by {{I-D.scion-dp}}.
 
 The process to look up and fetch path segments consists of the following steps:
 
 1. First, the source endpoint queries the control service in its own AS (i.e., the source AS) for the required segments. The control service has up-path segments stored in its path database. Additionally, the control service checks if it has appropriate core- and down-path segments in store as well; in this case it returns them immediately.
 2. If there are no appropriate core-segments and down-segments, the control service in the source AS queries the control services of the reachable core ASes in the source ISD, for core-path segments to core ASes in the destination ISD (which is either the own or a remote ISD). To reach the core control services, the control service of the source AS uses the locally stored up-path segments.
 3. Next, the control service of the source AS combines up-path segments with the newly retrieved core-path segments. The control service then queries the control services of the remote core ASes in the destination ISD, to fetch down-path segments to the destination AS. To reach the remote core ASes, the control service of the source AS uses the previously obtained and combined up- and core segments.
 4. Finally, the control service of the source AS returns all retrieved path segments to the source endpoint.
-5. Once it has obtained all path segments, the endpoint combines them into an end-to-end path in the data plane.
+5. Once it has obtained all path segments, the source endpoint combines them into an end-to-end path in the data plane.
+6. The destination endpoint, once it receives the first packet, may revert the path in the received packet to respond without needing to perform a further path lookup. This also ensures that traffic flows on the same path bidirectionally.
```

Comment:
Point 6: haven't we already explained that? May be we could be more laconic here:

6. The destination endpoint, once it receives the first packet, has the option to revert the path in the received packet in order to construct a response.

Reply:
I was a b it redundant with wording because of the feedback we received: 

> think in the provider world it is expected that traffic between two endpoints uses bidirectionally the same path symmetrically. As each AS can select its own path to other AS's, it looks like SCION is not symmetrical. Maybe a comment around that could help. 

I'll use your proposal for more succinct wording